module/move/willbe/src/command/readme_headers_renew.rs

```diff
@@ -0,0 +1,118 @@
+mod private
+{
+  use std::fmt::{Display, Formatter};
+  use error_tools::{ Result, err };
+  use crate::_path::AbsolutePath;

```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)